proto/lighthouse-response.proto

```diff
@@ -0,0 +1,363 @@
+syntax = "proto3";
+
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+// The overarching Lighthouse Response object (LHR)
+// see ../typings/lhr.d.ts
```

Comment:
WDYT would a github link to master with line number be easier, or I suppose they will always have to change in tandem?

Reply:
I can get behind linking it to the ts file in general, like `https://github.com/GoogleChrome/lighthouse/blob/master/typings/lhr.d.ts
`